# Finance E-Commerce Dataset



In [8]:
import pandas as pd

In [9]:
file = "/content/drive/MyDrive/Tugas-Tugas Praktikum/Tugas AVD/CheckPoint 1: Data Gathering & Business Understanding/Finance_Ecommerce_Dirty_Dataset_csv.csv"
df = pd.read_csv(file)
df.head()


TransactionID        Date AccountID   AccountName TransactionType  \
0         TX103251    4/3/2025    AC3973  Arjun Sharma           Debit   
1         TX101874   10/8/2024    AC8970     Dev Kumar           Debit   
2         TX103746   9/14/2024    AC3777    Amit Gupta           Debit   
3         TX105269  12/21/2024    AC9635      Ritu Roy          Refund   
4         TX100463   4/14/2024    AC1329      Raj Bose          Refund   
...            ...         ...       ...           ...             ...   
6055      TX105699  10/16/2023    AC5214   Vikram Iyer          Credit   
6056      TX102550   7/25/2025    AC8982   Anita Reddy           debit   
6057      TX100537   9/24/2023    AC5308   Rohit Kumar          Credit   
6058      TX101220   3/25/2024    AC7593    Sana Reddy          Credit   
6059      TX104271    2/3/2025    AC4810    Neha Reddy          Credit   

      Amount Currency  ExchangeRate         Balance      Merchant  ...  \
0      47.36     USD        80.6864     -1714968.59      ShopEasy  ...   
1     159.55      inr        1.0000     -1162428.75      AutoZone  ...   
2       2.43      GBP      102.0148     -2085783.34     BigBasket  ...   
3      78.96      AED       18.1216     -2727243.49      ShopEasy  ...   
4      42.81      USD       78.4379      -259175.06     LocalMart  ...   
...      ...      ...           ...             ...           ...  ...   
6055   69.09      USD       76.4757  ?-2,901,016.18      AutoZone  ...   
6056    4.94      USD       79.1632     -1557558.39      ShopEasy  ...   
6057   20.87      AED       19.7547      -276412.59        Amazon  ...   
6058    7.96      USD       76.8570      -593859.37      Reliance  ...   
6059   17.66      INR        1.0000     -2254562.37  ElectroWorld  ...   

      Subcategory    Country       City PostalCode           CardNumber  \
0           Books         UK       Pune   153621.0  XXXX-XXXX-XXXX-4249   
1         Concert  Australia      Delhi   380312.0       XXXX-XXXX-5979   
2       Specialty        USA   Gurugram        NaN       XXXX-XXXX-2243   
3      Restaurant  Australia     Mumbai   793747.0       XXXX-XXXX-9279   
4        FastFood        USA     Mumbai   755067.0                  NaN   
...           ...        ...        ...        ...                  ...   
6055        Hotel        UAE       Pune   856227.0       XXXX-XXXX-6489   
6056      Concert  Australia  Bengaluru   235120.0       XXXX-XXXX-5540   
6057     Pharmacy        UAE     Mumbai   263866.0  XXXX-XXXX-XXXX-6774   
6058          Men      India  Hyderabad   152222.0       XXXX-XXXX-7745   
6059    Accessory         UK    Chennai   347582.0  XXXX-XXXX-XXXX-9831   

                         Email         Phone IsFraud                 Notes  \
0     arjun.sharma@hotmail.com -4.469492e+09      No       repeat purchase   
1          dev.kumar@yahoo.com  5.882006e+09      No     cashback credited   
2       amit.gupta@example.com  8.832898e+08      No     cashback credited   
3         ritu.roy@hotmail.com  6.845400e+09      No       suspected fraud   
4         raj.bose@hotmail.com  1.228611e+09      No     cashback credited   
...                        ...           ...     ...                   ...   
6055     vikram.iyer@yahoo.com  4.081945e+09      No  chargeback requested   
6056     anita.reddy@gmail.com  2.586561e+09      No  monthly subscription   
6057   rohit.kumar@hotmail.com           NaN      No                   NaN   
6058      sana.reddy@yahoo.com  6.866496e+09      No     manual adjustment   
6059    neha.reddy@example.com  1.500553e+09      No  monthly subscription   

      CustomerSince  
0        16/09/2018  
1         11/7/2022  
2            Aug-21  
3              2024  
4          3/2/2023  
...             ...  
6055           2021  
6056      5/27/2023  
6057        unknown  
6058         Aug-19  
6059     20/06/2020  

[6060 rows x 23 columns]

# Business Understanding



## Business Objective

Dataset ini merupakan data-data transaksi keuangan pada platform e-commerce. Dataset ini memiliki 6060 baris x 23 kolom.

Pada dasarnya setiap kebutuhan berbisnis didalam dunia digital diperlukan adanya sebuah analisis terhadap kebutuhan pasar, akan hal tersebut kita perlu melihat dimana dan apa kategori pasar yang cocok untuk terjun kedalam industri tersebut, agar dapat hasil terbaik saat menjalankan bisnis e-commerce

Dataset ini akan digunakan yaitu, sebagai berikut:
1. Menganalisis tren transaksi dari waktu ke waktu
2. Mengidentifikasi kategori dan subkategori pilihan pasar
3. Menganalisis transaksi berdasarkan lokasi pelanggan

## Assess Situation
Platform e-commerce setiap harinya melakukan ribuan transaksi, yang dimana setiap data transaksi menyimpan informasi penting bagi pengambilan keputusan

Dengan analisis data-data transaksi, perusahaan-perusahaan dapat memahami pola belanja pelanggan, dan akan hal tersebut sangat membantu bisnis memahami kondisi keuangan dan pola pasar konsumen. Berikut merupakan ciri-ciri dari dataset:

1. Sumber Data  
Dataset Finance_Ecommerce_Dirty_Dataset dari Kaggle.

2. Kondisi Data
* 6061 baris  
* 23 kolom  
* Terdapat data kotor dan nilai kosong

3. Kendala
* Data-data ada yang kosong


## Analytic Goals & Project Plan

### Tujuan
1. Analisis tren transaksi dari waktu ke waktu
2. Analisis kategori dan subkategori pilihan pasar
3. Analisis transaksi berdasarkan lokasi pelanggan

### Rencana Proyek
1. Memahami Data  
2. Mengelola dan membersihkan data  
3. Visualisasi data
4. Menyusun rapi hasil data
4. Membuat ringkasan dan kesimpulan